apps/framework-cli/src/cli.rs

```diff
@@ -1008,21 +1008,85 @@ pub async fn cli_run() {
 
     info!("CLI Configuration loaded and logging setup: {:?}", config);
 
-    let cli = Cli::parse();
+    // Setup Ctrl+C / SIGINT handler
+    tokio::spawn(async move {
```

Comment:
Maybe why do we need this altogether? the long running process is only when we have the webserver running, which we already have a hook for shutdown there. I dont' think this file changes are necessary

Reply:
It's an attempt to ensure there's coverage during shutdown.  Will remove and test to confirm.